src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -453,10 +453,11 @@ public void menuItemValidateClicked() {
    * changes, and creates and launches TabulatorService from the saved config path.
    */
   public void menuItemTabulateClicked() {
-    if (checkForSaveAndExecute()) {
+    String filepath = commitConfigToFileAndGetFilepath();
```

Comment:
> I opted to have this, which has a pretty nice UX to me

Agree that it's a cool idea, but I think it poses a problem in that we need to ensure that the config file exactly matches what was actually used to run a tabulation. If we could make this pop-up _only_ appear if the version is `TEST`, that would be more palatable I think.

Reply:
Perhaps this is the solution to what I mentioned above: if version is `TEST`, then it follows this workflow. If it's anything else, it reverts to the old workflow (`save` or `cancel`).

Reply:
Yep, that sounds good to me.

Reply:
Alright, done!